appengine/myria_web_main.py

```diff
@@ -631,15 +539,27 @@ def __init__(self, debug=True,
             ('/execute', Execute),
             ('/dot', Dot),
             ('/examples', Examples),
-            ('/demo3', Demo3)
+            ('/demo3', Demo3),
+            ('/page', MyriaPage)
         ]
 
         # Connection to Myria. Thread-safe
-        self.connection = myria.MyriaConnection(
-            hostname=hostname, port=port, ssl=ssl)
-        self.hostname = hostname
-        self.port = port
+        self.myriaConnection = myria.MyriaConnection(hostname=hostname,
+                                                     port=port, ssl=ssl)
+        self.myriahostname = hostname
+        self.myriaport = port
         self.ssl = ssl
+        self.clanghostname = 'localhost'
```

Comment:
See #268 for one idea about how we can separate configuration from implementation.